README.rst

```diff
@@ -268,6 +268,8 @@ Scrapers available for:
 - `https://www.maangchi.com <https://www.maangchi.com>`_
 - `https://madensverden.dk/ <https://madensverden.dk/>`_
 - `https://madsvin.com/ <https://madsvin.com/>`_
+- `https://marleyspoon.de/ <https://marleyspoon.de/>`_ (*)
+    - `.at <https://marleyspoon.at/>`__, `.nl <https://marleyspoon.nl/>`__, `.be <https://marleyspoon.be/>`__, `.com <https://marleyspoon.com/>`__, `.com.au <https://marleyspoon.com.au/>`__
```

Comment:
To appease the unit tests, we'll need to make two adjustments related to the domain name list:

  * Listing the nested TLDs in lexographically sorted order (`.at, .be, .com, .com.au, ...`)
  * Registering the corresponding entries in the `SCRAPERS` dictionary (`MarleySpoon.host(domain=at): MarleySpoon,` - it's a mapping of hostnames to implementation classes, essentially)

Reply:
Thanks for explaining all your suggestions so well! I really appreciate it.
If the scrapers dictionary filters by domain, does that make the locale exception obsolete? Would it ever be raised? If someone tries to scrape marleyspoon.es, wouldn't they be told that there's no scraper available?

Reply:
You're welcome!  No, I think the locale exceptions are still relevant despite the per-domain-name scraper selection.  In particular, that's because the domain name (`.es`, for example) doesn't always imply what the language of the content in the response HTML will be -- especially if the user configures `Accept-Language` HTTP headers to a different language.

Reply:
Ah, perhaps I should have chosen a different example than `.es` since you used that too :)

Basically: you're correct that if the user tries to scrape an unsupported domain name, then no custom scraper will be returned.  However, even if a scraper _is_ found, it that doesn't guarantee anything about the language of the HTML's content, if that makes sense.